scraper/src/v2_json_downloader.py

```diff
@@ -0,0 +1,50 @@
+import urllib.request
+import json
+import os
+import ssl
+
+
+def generate_v2_file(url, data, name=""):
+    life_file_name_v2 = url.split("/")[-1].split(".")[0] + "_v2.json"  # generating new file name from
+    if name != "":
+        life_file_name_v2 = name + "_v2.json"
+
+    f = open("../../data/" + life_file_name_v2, 'w')
+    print("Adding file: "+life_file_name_v2)
+    f.write(json.dumps(data))
+    f.close()
+
+
+def parseOxygen(data):
```

Comment:
Can you generalise the parse function.

It can take in a key and set the flag 😅.

You can assume the key would be same as the file name in lower case.

Reply:
ok @bodhish , For oxygen what fields in oxygen.json determines it to be "true" in active_district_data